template/pkg/config.sls

```diff
@@ -20,3 +20,5 @@ template-config:
     - user: root
     - group: root
     - template: jinja
+    - require:
+      - sls: {{ tpldot }}.install
```

Comment:
It's a shame that relative includes (i.e. `.install`) can't be used when settings a requisite based on an `sls` -- this is an [upstream bug](https://github.com/saltstack/salt/issues/10838) that still hasn't been resolved.

Reply:
I didn't even know they existed, where did you find these...

Reply:
@aboe76 I've linked to the code in the [first comment](https://github.com/saltstack-formulas/template-formula/pull/48#issue-255703627).